decisions/why-material-design/index.qmd

```diff
@@ -0,0 +1,44 @@
+---
+title: "Why Material Design"
+description: "We want to Seedcase to look good with a minimal effort."
```

Comment:
Showcasing a demo of Seedcase software helps make things more tangible. This
```suggestion
description: "We aim to utilise existing CSS or UI frameworks instead of building from scratch so Seedcase will look good with minimal effort. Since Material Design lives up to these requirements, we have chosen this framework for Seedcase."
```